azureml/pipelines/configs/fewshot_knn_config.yaml

```diff
@@ -9,10 +9,12 @@ knn_fewshot_config:
     base_experiment_name: fewshot_knn
     tags:
     default_compute_target: isolatedcompute
-  mmlu_dataset: clinical_knowledge
+  mmlu_dataset: all_medicine_datasets
```

Comment:
I'd recommend using all MMLU datasets by default. Although this code originated on medical datasets, Promptbase reports numbers on the whole dataset, in particular for comparison with Gemini

Reply:
Will do